pages/offices/index.tsx

```diff
@@ -119,7 +120,7 @@ export default function OfficeIndex(
                 </ul>
                 <div className="hidden sm:block">
                   <MicrosoftPanel />
-                  <TestimonialPanel testimonial={props.testimonial} />
+                  <TestimonialPanel props={props.testimonial} />
```

Comment:
```suggestion
                  <TestimonialPanel{...props.testimonial} />
```

Reply:
Not done - as per reason mentioned above.